TimeLog.Api.Documentation/Source/Reporting/GetProjectsRaw.xsd

```diff
@@ -67,6 +67,7 @@
       <xsd:element name="CurrencyISOCode" type="xsd:string" />
       <xsd:element name="CurrencyRate" type="xsd:decimal" />
       <xsd:element name="IsMultiCustomerProject" type="xsd:boolean" />
+      <xsd:element name="LegalEntityID" type="xsd:positiveInteger" />

```

Comment:
Please add `minOccurs` and `maxOccurs` for completeness.

Reply:
It's added.